### PR TITLE
Flip over to new domains db

### PR DESCRIPTION
**Pull request by esoergel**
To be merged after successful deploy and processing of https://github.com/dimagi/commcare-hq/pull/9172

### CODE REVIEW

**Comment by czue** [comment]
assume you'll plug away until tests pass

**Comment by esoergel** [comment]
Yup, I think tests will fail at least until this branch has the other cleanup branches incorporated.

File: corehq/couchapps/by_domain_doc_type_date/views/view/map.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+function (doc) {
```
Comment (dimagimon) [inline corehq/couchapps/by_domain_doc_type_date/views/view/map.js]: Missing name in function declaration.

**Comment by dannyroberts** [comment]
only relevant part of the build failure is:
```
-  u'couchdb_uri': u'http://127.0.0.1:5984/commcarehq_test',
?  -               -
+  'couchdb_uri': 'http://127.0.0.1:5984/commcarehq__domains_test',
?                                                   +++++++++
```

**Comment by esoergel** [comment]
killing last test 'cause all I did was change the es index name.  This should be g2g.

**Comment by esoergel** [comment]
:scream_cat:

**Comment by dannyroberts** [comment]
:tada: :scissors:

**Comment by czue** [comment]
:dancer: :balloon: :fireworks:

**Pull request merged**